services/infrastructure/src/nservicebus/artiso.AdsdHotel.Infrastructure.NServiceBus.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="NServiceBus" Version="7.4.4" />
```

Comment:
_tabs_

Reply:
only for xml files or everywhere?

Reply:
I'd use spaces everywhere: having a mix of them has no advantage, and tabs look weird on markdowns/github.

Even further, I personally use 2 spaces for csproj files, since that seems to be the default indentation of visual studio.
![image](https://user-images.githubusercontent.com/8939890/98227491-ac675100-1f57-11eb-8932-757ac1281748.png)

Reply:
I've added an editorconfig file. Spaces for XML and C#. XML with 2 spaces, C# with 4 spaces indentation.